addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js

```diff
@@ -106,37 +159,54 @@ QUnit.test('click on "add followers" button', async function (assert) {
     });
     const bus = new Bus();
     bus.on('do-action', null, ({ action, options }) => {
-            assert.step('action:open_view');
-            assert.strictEqual(
-                action.context.default_res_model,
-                'res.partner',
-                "'The 'add followers' action should contain thread model in context'"
-            );
-            assert.strictEqual(
-                action.context.default_res_id,
-                resPartnerId1,
-                "The 'add followers' action should contain thread id in context"
-            );
-            assert.strictEqual(
-                action.res_model,
-                'mail.wizard.invite',
-                "The 'add followers' action should be a wizard invite of mail module"
-            );
-            assert.strictEqual(
-                action.type,
-                "ir.actions.act_window",
-                "The 'add followers' action should be of type 'ir.actions.act_window'"
-            );
-            pyEnv['res.partner'].write([action.context.default_res_id], { message_follower_ids: [mailFollowerId1] });
-            options.on_close();
+        assert.step('action:open_view');
```

Comment:
don't change indent here, it was done on purpose in preparation of following changes

Reply:
Ho ok !